packaging/hudi-flink-bundle/pom.xml

```diff
@@ -430,6 +430,12 @@
       <version>${flink.version}</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.flink</groupId>
+      <artifactId>flink-avro</artifactId>
+      <version>${flink.version}</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
Why this change ?

Reply:
HoodieClusteringGroup is a avro model in ClusteringPlanEvent.

Reply:
But you should not depend on flink-avro i guess, there are already many model clazzes in hudi-flink code paths.

Reply:
fixed. use ClusteringGroupInfo instead of  HoodieClusteringGroup.